code/API_definitions/number-verification.yaml

```
@@ -2,43 +2,66 @@ openapi: 3.0.3
-    It also makes it possible for a Service provider to verify the number itself by returning the phone number associated to the authenticated user's access token.
+    API to verify or retrieve the **mobile phone number** that is currently used on the user's device
 
-    In this API **phone number** term refers to the mobile phone number.
+    In this API **phone number** refers to the mobile phone number.
 
     # Introduction
 
-    Number Verification API performs real-time checks to verify the phone number of the mobile device being used to access a service provider (SP) service, where the mobile device is accessing the *service provider* over a mobile network (WiFi connections are out of this API scope) either by getting the comparison result or receiving the phone number of the device that it is used, so they can verify it themselves.
-
-    It uses direct mobile network connections to verify possession of a phone number in the background without requiring user interaction. There are neither OTPs (One-time passwords) received by SMS nor authenticator app downloads, so it is much simpler. It can be used at sign up, login, or transaction time to validate that a user's SIM (Subscriber Identity Module) is both actively connected to the mobile network and not spoofed or cloned.
+    Number Verification API performs real-time checks to verify the phone number of the mobile device being used to access a service provider (SP) service.
 
+    The mobile phone number is personal data and end-user consent might be required.
 
     # Relevant Definitions and Concepts
 
-    - **Network-Based Authentication**: Authentication mechanism based on the identification of the endpoint of a network connection. Network operators know to which subscriber a network resource is assigned at a given moment, for example the mobile phone number associated to a specific mobile network connection.
-
+    - **Network-Based Authentication**: Authentication mechanism based on the identification of the endpoint of a network connection.
+    Network operators know to which subscriber a network resource is assigned at a given moment, for example the mobile phone number associated to a specific mobile network connection.
+    - **SIM-Based Authentication**: Authentication mechanism based on the identification of the subscriber's SIM installed in the user's device.
     # API Functionality
 
-    This enables a Service Provider (SP) to verify the phone number of the mobile device being used to access their service where the mobile device is accessing the *service provider* over a mobile network (WiFi connections are out of this API scope). This can happen either by getting the comparison result or receiving the phone number of the device that is used, so they can verify it themselves.
+    This API enables an API Consumer to verify or retrieve the phone number of the mobile device being used to access their service
```

We can keep the original removing the reference to mobile network auth

```suggestion
    This enables a Service Provider (SP) to verify the phone number of the mobile device being used to access their service. This can happen either by getting the comparison result or receiving the phone number of the device that is used, so they can verify it themselves.
```

---

I am in favor of some rewording. Please provide suggestions.
Or create a quick rewording PR...
If we merge one PR, we can merge two.

---

As this is something done in other Release reviews, it makes sense to do the same. 
Rewording PR decoupled from this one would be fine, seems to be more work. I'd support both approaches

---

I think there is more support for getting rid of "Service Provider" than to keep it.
So, I am closing this conversation.

If you think this is a blocker to getting this merged, then please create a new comment.